src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -179,6 +185,7 @@ class BaseCreateCourseForm extends React.Component {
 BaseCreateCourseForm.defaultProps = {
   isCreating: false,
   pristine: true,
+  currentValues: undefined,
```

Comment:
I don't think `undefined` qualifies as a shape, which is how you define this prop. If you defaulted to `{}`, you could also probably drop the `currentValues &&` check above?